### ISSUE TITLE
TODO:  Password forms should have (optionally hidden) username fields for accessibility

### ISSUE THREAD

**Issue by ArpadGBondor** [ArpadGBondor/Leave-Management#107]
<img width="1140" height="178" alt="Image" src="https://github.com/user-attachments/assets/669fa92e-925d-4a43-894f-a87ba01283f1" />

Add Password / Update Password forms
